docs/design-docs/reporting-sot.md

```diff
@@ -0,0 +1,538 @@
+schema in-person and came up with the following:
+
+| method | year | geography_type | geography_id | geography_data_year | group_type     | group_id | stage_name | stage_id | additional_cols... |
+|--------|------|----------------|--------------|---------------------|----------------|----------|------------|----------|--------------------|
+| v1     | 2023 | municipality   | Chicago      | 2022                | class          | 204      | board      | 3        | ...                |
+| v1     | 2010 | triad          | 1            | 2010                | modeling_group | SF       | mailed     | 1        | ...                |
+
+### Primary key columns
+
+- `method (string)` - The method version number used for lifecycle
+  management. Hopefully set to `v1` forever, but more likely to increment for
+  each breaking change
+- `year (string)` - Tax year. Should derive existence from `iasworld.pardat`
+- `geography_type (string)` - The type of geography the stat is grouped by,
+  e.g. municipality, triad, township, etc. Should encompass all geographies
+  currently in our PIN universe
+- `geography_id (string)` - The identifier for each unit of geography,
+  e.g. for municipality this would be `Chicago`, for township this might be
+  "Thornton"
+- `geography_data_year (string)` - The year of the file the geography
+  definition is sourced from e.g. a 2020 Census boundary would be `2020`
+- `group_type (string)` - The sub-group for aggregation within each geometry
+  e.g. class, modeling group, etc.
+- `group_id (string)` - The sub-group identifier e.g. for class it would be
+  `202`, `203`, etc.
+- `(optional) stage_name (string)` - Name of the assessment stage used for
+  grouping, one of `mailed`, `certified`, or `board`
+- `(optional) stage_id (string)` - Numeric identifier of the assessment
+  stage, one of `1` (mailed), `2` (certified), `3` (board)
+
+#### Enumerated geography_type
+
+The `geography_type` column above should span the following values. Where both
+names and codes are available, we should prefer codes e.g. `1` instead of
+`City` for triad.
+
+- `county` (no grouping, all of Cook County)
+- `triad`
+- `township`
+- `nbhd` (Assessor codes)
+- `tax_code`
+- `zip_code`
+- `community_area` (if available)
+- `census_place`
+- `census_tract`
+- `census_congressional_district`
+- `census_zcta`
+- `cook_board_of_review_district`
+- `cook_commissioner_district`
+- `cook_judicial_district`
+- `ward_num` (if available)
+- `police_district` (if available)
+- `school_elementary_district`
+- `school_secondary_district`
+- `school_unified_district`
+- `municipality`
+- `tax_park_district`
+- `tax_library_district`
+- `tax_fire_protection_district`
+- `tax_community_college_district`
+- `tax_sanitation_district`
+- `tax_special_service_area`
+- `tax_tif_district`
+- `central_business_district`
+
+#### Enumerated group_type
+
+The `group_type` column above should span the following values.
```

Comment:
@ccao-jardine @wrridgeway Any other groups you all can think of?

Reply:
No, this covers them.